backend/src/packages/chat-messages/chat-message.service.ts

```diff
@@ -56,10 +58,14 @@ class ChatMessageService implements Service {
       chatId,
     );
 
+    let group: ChatMessagesGroups = {};
+    chatMessages.forEach((chatMessage) => {
+      const chatData = chatMessage.toObject();
+      group = groupChatMessage(group, chatData);
+    });
```

Comment:
Can we use .reduce instead?

Reply:
@v0ldemar01 No, it's forbidden by the linter

Reply:
Okey, just disable it)

Reply:
Done